production/wazuh/cluster/wazuh_cf_master.sh

```diff
@@ -131,6 +131,12 @@ cat >> ${manager_config} << EOF
     <use_source_ip>no</use_source_ip>
     <force_insert>yes</force_insert>
     <force_time>0</force_time>
```

Comment:
These options were depreciated. It would be necessary to remove them from the ossec template